app/controllers/devices_guidance_controller.rb

```diff
@@ -1,4 +1,6 @@
 class DevicesGuidanceController < ApplicationController
+  include Rails.application.routes.url_helpers
```

Comment:
is this line redundant? I've run the specs  & clicked through with it commented-out, and it all seems to work

Reply:
Yes, you're right - it's not needed. I've removed it.